### PR TITLE
Added Location header to run creation

### PR DESCRIPTION
**Pull request by nkijak**
Fix for #699

### CODE REVIEW

**Comment by nkijak** [comment]
First time using JAX-RS, so not sure if this is the best way to get the location header.  Didn't want to hardcode the value but the reflection step to get the method seems like it might be bad.

File: src/main/java/marquez/service/models/Run.java [hunk 1]
```diff
@@ -33,6 +33,10 @@
   @NonNull Run.State state;
   @Nullable Map<String, String> args;
 
+  public UUID getId() {
```
Comment (wslulciuc) [inline src/main/java/marquez/service/models/Run.java]: _Minor:_ [Lombok](https://projectlombok.org) will generate getters/setters for us. So, we don't need to explicitly define one.

**Comment by nkijak** [comment]
@wslulciuc made the update but the commits are messy. Should I close this and open another?

**Comment by wslulciuc** [comment]
@nkijak My PRs get messy as well. I recommend [squashing](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) your commits into one. Then, we can merge this sweet feature!

**Comment by nkijak** [comment]
@wslulciuc squashed

**Pull request merged**